src/commands/lambda/__tests__/instrument.test.ts

```diff
@@ -493,12 +493,9 @@ TagResource -> arn:aws:lambda:us-east-1:123456789012:function:lambda-hello-world
         mockGitStatus.mockImplementation(() => ({
           ahead: 0,
           hash: '1be168ff837f043bde17c0314341c84271047b31',
+          remote: 'git.repository_url:git@github.com:datadog/test.git',
```

Comment:
Can we confirm two things here?

1. All tracers that we use support colon values in their tag values. I think just a quick message to the source code integration team would work here. 

2. That this change is backwards compatible with functions running older versions of a layer? I'm thinking of a situation where a customer has an old version of the lambda tracer that doesn't support the colon in the tag value, and this adds the tag which could cause errors in the application. If datadog-ci also installs the latest layers then I don't think this is an issue.

Reply:
I'd actually also check to make sure that datadog-cdk-constructs and serverless-plugin-datadog use pinned versions of this dependency because I know that was an issue for us before and it actually broke some of the deployments for our customers.

Reply:
Writing what we discussed here:

- All tracer versions definitely do not support colons, this would break backwards compatibility with those tracers. Add logic to format the git remote to turn `git@github.com:datadog/test.git` and `https://github.com/datadog/test.git` into `github.com/datadog/test.git`
- datadog-cdk-contructs and serverless-plugin-datadog both use `uploadGitCommitHash` from the `git-metadata` library, which this Pr doesn't touch. So we're good for that